cds-plugin.js

```diff
@@ -0,0 +1,22 @@
+// https://pages.github.tools.sap/cap/docs/node.js/cds-plugins
+"use strict";
+
+const cds = require("@sap/cds");
+const { initializeFeatures } = require("./src/singleton");
+
+const activate = async () => {
+  if (cds.env.featureToggles && cds.env.featureToggles.plugin) {
+    cds.env.requires = Object.assign(cds.env.requires, {
+      "@cap-js-community/feature-toggle-library": { model: "@cap-js-community/feature-toggle-library" },
+    });
```

Comment:
@soccermax FYI I think this is a better way to load any model plugins might need. It ties the model loading to the plugin being enabled. If you do it in `package.json` the model loading would always happen, even if the plugin was disabled.